cognite-sdk-dotnet.sln

```diff
@@ -81,18 +77,6 @@ Global
 		{A94F7DFA-2202-4705-AADB-FBBF739BED61}.Release|x64.Build.0 = Release|Any CPU
 		{A94F7DFA-2202-4705-AADB-FBBF739BED61}.Release|x86.ActiveCfg = Release|Any CPU
 		{A94F7DFA-2202-4705-AADB-FBBF739BED61}.Release|x86.Build.0 = Release|Any CPU
-		{430F4C09-1FB5-4674-987B-AAFAC8AA25DE}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
maybe we should add this new "Examples" project back to the solution?

Reply:
Good catch! I added back the Examples to the solution.